src/test/java/io/appium/java_client/service/local/ServerBuilderTest.java

```diff
@@ -54,6 +57,9 @@
     private static final Path PATH_T0_TEST_MAIN_JS = ROOT_TEST_PATH
             .resolve("service").resolve("local").resolve("main.js");
 
+    private static final String INVALID_BASE_PATH_ERROR_MESSAGE =
```

Comment:
it is usually ok to just verify the exception class. Tests become too flaky because string messages could be changed too often

Reply:
Updated in the new commit, thanks for highlighting